src/core/util/xlio_stats.h

```diff
@@ -337,30 +345,38 @@ static const char *const ring_type_str[] = {"RING_ETH", "RING_TAP"};
 typedef struct {
     uint64_t n_rx_pkt_count;
     uint64_t n_rx_byte_count;
+    uint64_t n_rx_interrupt_requests;
+    uint64_t n_rx_interrupt_received;
+    uint32_t n_rx_cq_moderation_count;
+    uint32_t n_rx_cq_moderation_period;
+    uint32_t n_rx_tls_contexts;
+    uint32_t n_rx_tls_resyncs;
+    uint32_t n_rx_tls_auth_fail;
+    ring_type_t n_type;
+    void *p_ring_master;
+
+    // First cache line
```

Comment:
I'd suggest to add a compile time assert to guarantee the cache line alignment (by field offset or sizeof of the inner struct)

Reply:
I added static assert outside the class on the offset as it does not recognize the class from within.